charters/DataOperations/charter.md

```diff
@@ -0,0 +1,89 @@
+  - Monitor the status of data submissions at each step in the DCP data workflow from initial identification of suitability through data release and potential inclusion in additional data distribution
+  - Engage with Governance Group and Analysis Work Group to identify data quality standards which dictate data eligible to be released and data eligible to be included in data distributions
+  - Engage with compliance to ensure that the data release and data distribution processes are facilitating appropriate data access to all of the types of consumers
+  - Ensure released data are useable to tertiary portals and users can easily cross-reference across portals
+  - Determine best practices for when and how changes are made to DCP outputs after release
+
+- Community engagement
+  - Define data use policy and citation instructions for HCA data users
+  - Define mechanism(s) for which the community is notified when DCP products are updated after release
+  - Partner with Content and Portal Development to ensure clear, findable, and current documentation describing data release and data distribution
+  - Ensure effective communication of HCA data updates and user features to the scientific community
+  - Partner with UX to assess curation added value of data distribution through communities’ feedback, and better understand their expectations and needs
+  - Onboard staff to DCP ZenDesk ticketing system
+  - Take ownership of establishing process and interfacing with DCP components for assigning, responding, and integrating ZenDesk tickets
+  - Compile feedback gained from consumers through ZenDesk queries and comments, and report back to the DCP in order to facilitate system improvement 
+
+
+
+### Scientific "guardrails" [Optional]
+
+In order to meet the needs of the communities of scientific consumers, the DataOps team requires input from the Governance Group as to where curated data distribution will add value for the researcher.
+
+## Out-of-scope
+
+The Data Operations is not responsible for determining what data or data types are made available through the HCA.
+
+As many of the features and requirements for data release and data distribution will be implemented through other components, the majority of technical aspects pertaining to data release and data distribution will be implemented by the appropriate component, and not Data Operations.
+
+## Milestones and Deliverables
+
+2019 Q3
+- Follow a dataset through the entire data workflow in order to understand each step and assess the current process. 
+- Organize incentivized data submission program in conjunction with the HCA General Meeting in Barcelona in order to encourage pre-publication data submission. 
+- Establish ZenDesk ticket triage procedure.
+
+2019 Q4
+- Write actional RFCs aimed to improve the data workflow.
+
+2020 Q1
```

Comment:
Added 'Issue the first HCA Snapshot' to the Deliverables for 2020 Q1 in 0de0c59

Reply:
For a cadence, it may make since to set a 'safe' gap between v1 and v2. This will allow us to be realistic in terms of DCP-wide bandwidth required to put one of these out, and also gives us more time to adjust to community feedback for v2. So maybe that's 6 months. Then as we get the process more streamlined (and data flow picks up), we may shift towards quarterly.